packages/api/src/external/carequality/document/query-documents.ts

```diff
@@ -0,0 +1,74 @@
+import { makeLambdaClient } from "@metriport/core/external/aws/lambda";
+import { errorToString } from "@metriport/core/util/error/index";
+import { capture } from "@metriport/core/util/notifications";
+import { Patient } from "../../../domain/medical/patient";
+import { Config } from "../../../shared/config";
+import { createCQDocumentQueryRequest } from "./document-query-request";
+import { getOrganizationOrFail } from "../../../command/medical/organization/get-organization";
+import { makeIheGatewayAPI } from "../api";
+import { MedicalDataSource } from "../../../external";
+import { getCQPatientData } from "../command/cq-patient-data/get-cq-data";
+import { appendDocQueryProgressWithSource } from "../../hie/append-doc-query-progress-with-source";
+
+const region = Config.getAWSRegion();
+const lambdaClient = makeLambdaClient(region);
+const iheGateway = makeIheGatewayAPI();
+const lambdaName = Config.getDocQueryResultsLambdaName();
+
+export async function getDocumentsFromCQ({
+  requestId,
+  patient,
+}: {
+  requestId: string;
+  patient: Patient;
+}) {
+  if (!iheGateway) return;
+
+  try {
+    const organization = await getOrganizationOrFail({ cxId: patient.cxId });
+    const cqPatientData = await getCQPatientData({ id: patient.id, cxId: patient.cxId });
+
+    const documentQueryRequest = createCQDocumentQueryRequest({
+      requestId,
```

Comment:
nit: 
const documentQueryRequests = createCQDocumentQueryRequests
since function is creating many requests